### PR TITLE
refresh from OpenLineage/docs

### PR DESCRIPTION
**Pull request by pawel-big-lebowski**
Sync OpenLineage/docs content with OpenLineage/Openlineage repo and website/docs directory. 

PR is required as probably previous sync didn't fetch latest version available,

### CODE REVIEW

File: website/docs/releases/0_16_1.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ sidebar_position: 9975
 * Airflow: enforce column casing in `SQLCheckExtractor`s [`#1159`](https://github.com/OpenLineage/OpenLineage/pull/1159) [@denimalpaca](https://github.com/denimalpaca)  
     *Uses the parent extractor's `_is_uppercase_names` property to determine if the column should be upper cased in the `SQLColumnCheckExtractor`'s `_get_input_facets()` method.*
 * Spark: prevent exception when no schema provided [`#1180`](https://github.com/OpenLineage/OpenLineage/pull/1180) [@pawel-big-lebowski](https://github.com/pawel-big-lebowski)  
-    *Prevents evaluation of column lineage when the `schemaFacet` is `null`.*
+    *Prevents evaluation of column lineage when the `schemFacet` is `null`.*
```
Comment (JDarDagran) [inline website/docs/releases/0_16_1.md:34]: ?

File: website/docs/spec/object-model.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ A Run is an instance of a Job that represents one of its occurrences in time.
 
 Each run will have a uniquely identifiable `runId` that is generated by the client as [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier). The client is responsible for maintaining the `runId` between different Run State Updates in the same Run. It is recommended to use [UUIDv7](https://datatracker.ietf.org/doc/draft-ietf-uuidrev-rfc4122bis/) format.
 
-Runs can be used to observe changes in Jobs between their instances. If, for example, you have cron running a Python script that repeats a query every day, this should result in a separate Run for each day.
+Runs can be used to observe changes in Jobs between their instances. If, for example, you have cron running a Python script that repeats a query every day, this should resuilt in a separate Run for each day.
```
Comment (JDarDagran) [inline website/docs/spec/object-model.md:49]: ?

File: website/docs/releases/0_16_1.md (hunk 1 above)
Comment (pawel-big-lebowski) [reply website/docs/releases/0_16_1.md:34]: [    *Prevents evalution of column lineage when the `schemFacet` is `null`.*](https://github.com/OpenLineage/docs/blob/4722a53119c84cc62677fa5cab6ff1467cbb7b3a/docs/releases/0_16_1.md?plain=1#L34)

File: website/docs/spec/object-model.md (hunk 2 above)
Comment (pawel-big-lebowski) [reply website/docs/spec/object-model.md]: https://github.com/OpenLineage/docs/blob/4722a53119c84cc62677fa5cab6ff1467cbb7b3a/docs/spec/object-model.md?plain=1#L49

fixed

**Review by JDarDagran — APPROVED** [review]

**Pull request merged**